inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/sinks/SenderManager.java

```diff
@@ -264,11 +265,13 @@ public void sendBatchSync(String groupId, String streamId,
             if (result == SendResult.OK) {
```

Comment:
If `pluginSendCount` includes success and fail, is it better to add it just after doing send operation?